### PR TITLE
Update clusterLanding.html 

### PR DESCRIPTION
**Pull request by mokalebo**
Styling changes to filter search box.

### CODE REVIEW

**Comment by djw8605** [comment]
Your branch names are terrible!  😄  reviewing...

File: HCCGo/app/html/clusterLanding.html [hunk 1]
```diff
@@ -54,9 +54,12 @@ <h4 class="usage text-center">/work Usage</h4>
 
     <div class="col-md-8 jobdisplay">
       <div class="card">
-        <input type="search" ng-model="q" placeholder="filter jobs..." />
+        <div class="form-control">
+          <input type="search" class="form-control" style="width:150px" placeholder="Filter Jobs..." />
```
Comment (djw8605) [inline HCCGo/app/html/clusterLanding.html]: You still are not using the `ng-model="q"`, which you need!

File: HCCGo/app/html/clusterLanding.html [hunk 2]
```diff
@@ -54,9 +54,12 @@ <h4 class="usage text-center">/work Usage</h4>
...
+          <span class="help-block">Please enter a job</span>
         <a class="pull-right" href="javascript:void(0)" ng-click="refreshCluster(true)"><i id="jobrefresh" class="mdi-action-autorenew"></i></a>
         <hr />
+        </div>
```
Comment (djw8605) [inline HCCGo/app/html/clusterLanding.html]: The div is only for the input item.  You should move it right below the span.

File: HCCGo/app/html/clusterLanding.html (hunk 1 above)
Comment (djw8605) [inline HCCGo/app/html/clusterLanding.html]: Also, I'm not the biggest fan of inline styles...  But if it works, ok.

Comment (adamfitzgibbon) [inline HCCGo/app/html/clusterLanding.html]: Sorry for creepin', but yeah inline styles hurt me, as do unnecessary static widths. Bootstrap grid classing would be good here, perhaps?

**Comment by djw8605** [comment]
This is how it looks on my screen.  Is this how it looks on your screen?  It looks pretty screwed up.

image

**Comment by djw8605** [comment]
You may not be able to use `form-control`, or you might have to make sure it doesn't stretch across the top.

**Comment by mokalebo** [comment]
screen shot 2017-02-16 at 9 25 07 am
This is how it currently looks when I build on my machine. What causes the differences?

**Comment by djw8605** [comment]
Yeah, that still looks pretty weird.  I would remove the "Please enter a job" text.  Also, that line to the right of the search box should go below it.  Possibly you need to insert a `<br />` ?

**Comment by mokalebo** [comment]
screen shot 2017-02-16 at 9 50 36 am
I added the line break and took out the additional text at the bottom. How does this look?

**Comment by djw8605** [comment]
The little refresh button in the top right should be higher.   Is the `<br />` before or after that button?  It should be after.

**Comment by mokalebo** [comment]
screen shot 2017-02-16 at 9 59 29 am
It was before, I went ahead and changed it

**Comment by djw8605** [comment]
That looks great!  Try submitting a job, and see how the list looks when there is a job in the list.

**Comment by mokalebo** [comment]
screen shot 2017-02-16 at 10 16 29 am
This is how it looks now with jobs on the list.

**Comment by djw8605** [comment]
And the filter works and everything?

**Comment by mokalebo** [comment]
Yep, the filter works as well, I'll modify the pull request to reflect the changes I've made so that you can test

**Pull request merged**